digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/MassTransitStopService.scala

```diff
@@ -102,9 +102,11 @@ trait MassTransitStopService extends PointAssetOperations {
       return property
     val equipmentProperty = tierekisteriStop.equipments.filter(_._1.publicId == property.publicId)
     val mappedProperties = equipmentProperty.map {
-      case (equipment, existence) =>
-        PropertyValue(existence.propertyValue.toString, Some(massTransitStopDao.getPropertyDescription(property.publicId, existence.propertyValue.toString).head))
-    }
+      case (equipment, existence) if(equipment.isMaster) =>
+        Some(PropertyValue(existence.propertyValue.toString, Some(massTransitStopDao.getPropertyDescription(property.publicId, existence.propertyValue.toString).head)))

```

Comment:
We should avoid calling DAO in a loop, too much connection overhead, especially in batch runs.